src/Func.h

```diff
@@ -216,13 +216,60 @@ class Stage {
     EXPORT Stage &gpu(VarOrRVar block_x, VarOrRVar block_y, VarOrRVar block_z,
                       VarOrRVar thread_x, VarOrRVar thread_y, VarOrRVar thread_z,
                       DeviceAPI device_api = DeviceAPI::Default_GPU);
+
+    // TODO(psuriana): For now we need to expand "tx" into Var and RVar versions
```

Comment:
This seems dangerous... doesn't this mean that old gpu_tile APIs will transparently resolve to calls of the new interface? i.e. this would break an existing program that used the old API.